### PR TITLE
Updated troubleshooting docs

### LINKED ISSUE #260: Update troubleshooting guide to latest behavior
The existing [troubleshooting guide](https://notaryproject.dev/docs/troubleshooting/) is out-of-date, need to revisit each case and keep it updated. Maybe we should mention which release version the guide applies to.

### PR DESCRIPTION
**Pull request by Roseline-Bassey**
Fixing #260 

This PR includes modifications and updates to the following sections in the troubleshooting docs:

1. Updated: [How do I troubleshoot issues with signatures?](https://notaryproject.dev/docs/troubleshooting/#how-do-i-troubleshoot-issues-with-signatures)

2. [Modified: "Enabling notation CLI commands logging"](https://notaryproject.dev/docs/troubleshooting/#enabling-notation-cli-commands-logging)

3. [fixed missing closing parenthesis in this section](https://notaryproject.dev/docs/troubleshooting/#when-i-verify-an-artifact-i-get-the-error-error-open-homeconfignotationtrustpolicyjson-no-such-file-or-directory)

4. Included other types of trust stores in this[ section ](https://notaryproject.dev/docs/troubleshooting/#when-i-verify-an-artifact-i-get-the-error-homeconfignotationtruststorex509camytruststore-does-not-exist)

### CODE REVIEW

File: content/en/docs/troubleshooting.md [hunk 1]
```diff
@@ -12,91 +12,89 @@ To suggest a troubleshooting item, [create a pull request PR with your suggested
 
 ## How do I troubleshoot issues with signatures?
 
-Inspect the descriptor of a remote artifact sent for signing using  `oras manifest fetch`. For example:
+`oras manifest fetch` fetches information about an image; such as mediatype, config, layers, and annotations. By using the `--descriptor` flag, you can fetch only the descriptor of an artifact. Inspect the descriptor of a remote artifact sent for signing using `oras manifest fetch --descriptor`. For example: 
 
 ```console
-$ oras manifest fetch --descriptor IMAGE:TAG
+$ oras manifest fetch --descriptor REGISTRY/IMAGE:TAG
```
Comment (yizha1) [inline content/en/docs/troubleshooting.md:18]: I think IMAGE including the full URI and tag/digest, so maybe we can just use IMAGE

File: content/en/docs/troubleshooting.md [hunk 2]
```diff
@@ -12,91 +12,89 @@ To suggest a troubleshooting item, [create a pull request PR with your suggested
 
 ## How do I troubleshoot issues with signatures?
 
-Inspect the descriptor of a remote artifact sent for signing using  `oras manifest fetch`. For example:
+`oras manifest fetch` fetches information about an image; such as mediatype, config, layers, and annotations. By using the `--descriptor` flag, you can fetch only the descriptor of an artifact. Inspect the descriptor of a remote artifact sent for signing using `oras manifest fetch --descriptor`. For example:
```
Comment (yizha1) [inline content/en/docs/troubleshooting.md:15]: `oras manifest fetch` fetches information about an artifact, image is a type of artifact, there are other types of artifacts.

Comment (yizha1) [reply content/en/docs/troubleshooting.md:15]: And you use `artifact` in the sentence after the first sentence, so it's better to be consistent.

**Review by yizha1 — COMMENTED** [review]
Thanks @Roseline-Bassey left some comments.

**Comment by Roseline-Bassey** [comment]
Hi, @yizha1! Thanks a lot for your feedback. I've incorporated your suggestions into the new PR. Could you please take a look when you have a moment?

**Review by iamsamirzon — APPROVED** [review]

**Review by Two-Hearts — APPROVED** [review]
LGTM

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**